doc/src/Howto_rheo.rst

```diff
@@ -0,0 +1,115 @@
+Reproducing hydrodynamics and elastic objects (RHEO)
+====================================================
+
+The RHEO package is a hybrid implementation of smoothed particle
+hydrodynamics (SPH) for fluid flow, coupled to the :doc:`BPM package <Howto_bpm>`
+to model solid elements. RHEO combines these methods to enable mesh-free modeling
+of multi-phase material systems. The SPH solver supports many advanced options
+including reproducing kernels, particle shifting, free surface identification,
+and solid surface reconstruction. To model fluid-solid systems, the status of
+particles can dynamically change between a fluid and solid state, e.g. during
+melting/solidification, which determines how they interact and their physical
+behavior. The package is designed with modularity in mind, so one can easily
+turn various features on/off, adjust physical details of the system, or
+develop new capabilities. For instance, the numerics associated with
+calculating gradients, reproducing kernels, etc. are separated into distinct
+classes to simplify the development of new integration schemes which can call
+these calculations. Additional numerical details can be found in
+:ref:`(Palermo) <howto_rheo_palermo>` and
+:ref:`(Clemmer) <howto_rheo_clemmer>`.
+
+Note, if you simply want to run a traditional SPH simulation, the SPH package
```

Comment:
add link to Packages_details.rst for "SPH package"

Reply:
Also, is RHEO  + BPM related at all to the SMD package (same author) which as I recall does something SPH-like for elastic solids.  If so, it would be good to also link to SMD here and highlight the differences.

Reply:
My understanding of SMD is minimal, but I can add a mention.

Reply:
Added links to both packages.